content/concept-transaction-cost.md

```diff
@@ -19,7 +19,8 @@ Some transactions have different transaction costs:
 |-----------------------|--------------------------|
 | [Reference Transaction](#reference-transaction-cost) (Most transactions) | 10 drops |
 | [Key Reset Transaction](#key-reset-transaction) | 0 |
-| [Multi-signed transaction](reference-transaction-format.html#multi-signing) | 10 drops × (1 + Number of Signatures Provided) |
+| [Multi-signed Transaction](reference-transaction-format.html#multi-signing) | 10 drops × (1 + Number of Signatures Provided) |
+| [EscrowFinish Transaction with Fulfillment](reference-transaction-format.html#escrowfinish) | 10 drops × (32 + (Fulfillment size in bytes ÷ 16)) |
```

Comment:
Sorry, now I was wrong.
What you have is the calculation for how much extra is added to the base fee.
So the total is actually:
```
10 drops + (10 drops × (32 + (Fulfillment size in bytes ÷ 16)))
```
the second 10 drops is the `fee_ref`/`TRANSACTION_FEE_BASE`
https://github.com/ripple/rippled/blob/develop/src/ripple/core/Config.h#L135